src/sage/doctest/control.py

```diff
@@ -600,7 +601,12 @@ def _init_warn_long(self):
         if self.options.warn_long >= 0:     # Specified on the command line
             return
 
-        self.options.warn_long = 60.0
+        # The developer's guide says that even a "long time" test
+        # should ideally complete in under five seconds, so we're
+        # being rather generous here.
+        self.options.warn_long = 10.0
+        if self.options.long:
+            self.options.warn_long = 60.0
```

Comment:
Could we aim for 5.0 and 30.0? That's already 5x margin from what is recommended ( < 1s for normal tests / < 5s for long tests).

Are we too far from that? I don't think there are currently more than 50-100 tests that fail that and then it shouldn't be too hard to "fix" those (or just add `# long` or `# not tested` labels).

Reply:
Sure, I was trying not to ruffle any feathers, but I think 5s and 30s is still very reasonably. Done in a new commit. Ultimately it would be nice if we could achieve something like 1s/5s.